code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -54,7 +54,8 @@
 			if(prob(base_message_chance) && !suppress_warning)
 				to_chat(M, span_warning("[pick("You swallow excess mucus.", "You lightly cough.")]"))
 		else
-			M.emote("cough")
+			if(prob(20))
+				M.emote("cough")
```

Comment:
note that this serves as a tell for the airborne spreading

if that's fine with you, carry on

Reply:
Probably should just do an early return where `if(prob(80)) return` and then the rest of the code runs normally with the cough emote.  This would be more of a balance change but it would make sense.

Reply:
I don't wanna do that, Tim.